styles.css

```diff
@@ -0,0 +1,1537 @@
+  }
+
+  .menu-links {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: row;
+    top: 30px;
+    height: 100%;
+    width: 100vw;
+    z-index: 2;
+    margin-top: 10px;
+    color: #faf7f7;
+    font-family: "Lato", sans-serif;
+  }
+
+  .persons img {
+    width: 150px;
+    height: 150px;
+  }
+
+  .logo-about img {
+    width: 100px;
+    height: 50px;
+  }
+
+  .img-container img {
+    width: 300px;
+    height: 250px;
+    margin-left: 300px;
+  }
+
+  .logo2 a img {
+    width: 50px;
+    height: 50px;
+  }
+
+  .menu-links ul a img {
+    width: 50px;
+    height: 50px;
+  }
+
+  .mobile-nav {
+    display: none;
+  }
+
+  .menu-links a:hover {
+    color: rgb(237, 76, 18);
+  }
+
+  .menu-links ul {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    gap: 45px;
+    align-items: center;
+    width: 80%;
+  }
+
+  .menu-link:nth-child(1) {
+    margin-top: 10px;
+    margin-right: 400px;
+  }
+
+  .menu-link:nth-child(2) {
+    margin-top: 10px;
+  }
+
+  .menu-link:nth-child(3) {
+    margin-top: 10px;
+  }
+
+  .menu-link:nth-child(4) {
+    margin-top: 10px;
+  }
+
+  .menu-link:nth-child(5) {
+    margin-top: 10px;
+  }
+
+  .menu-link:nth-child(6) {
+    margin-top: 10px;
+  }
+
+  .menu-link:nth-child(7) li {
+    border: 5px solid rgb(237, 76, 18);
+    padding: 10px;
+    color: rgb(237, 76, 18);
+    flex-wrap: wrap;
+    margin-top: 10px;
+  }
+
+  .landing {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+    width: 100vw;
+    padding-top: 160px;
+    overflow-x: hidden;
+    background-repeat: no-repeat;
+    background-image: url(./images/bg.webp);
+    background-size: cover;
+    height: 100vh;
+    color: rgb(81, 80, 80);
+  }
+
+  .container {
+    width: 80%;
+    margin-left: 200px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .container h2 {
+    font-size: 2rem;
+    color: rgb(237, 76, 18);
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h1 {
+    font-size: 3rem;
+    color: rgb(237, 76, 18);
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container p {
+    padding: 1.4rem 3.4rem;
+    font-size: 1rem;
+    display: block;
+    margin: 2rem;
+    border: 2px solid #fff;
+    width: 690px;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h3 {
+    font-size: 2rem;
+    font-family: 'Lato', sans-serif;
+  }
+
+  .container h4 {
+    font-family: 'Lato', sans-serif;
+  }
+
+  .main {
+    background: url(./images/bgtexture.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+    overflow-x: hidden;
+    width: 100vw;
+    border-top: 1px solid #dddbdb;
+  }
+
+  .main-program {
+    display: grid;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-program h2 {
+    font-size: 2rem;
+    color: #fff;
+    font-family: 'Lato', sans-serif;
+  }

```

Comment:
- [ ] [OPTIONAL] As you add more css styling to your project, it is always best practice to go through the variours pages to ensure nothing is being destroyed or disorganised and it matches the original design :confused: . Kindly consider going through the following and fix any abnormallities in order to maintain the best optimum website. :wink: 

![Screenshot from 2023-01-04 23-45-45](https://user-images.githubusercontent.com/65233860/210672232-be628451-4900-44db-9c01-e026ff4c36ec.png)


![Screenshot from 2023-01-04 23-57-52](https://user-images.githubusercontent.com/65233860/210672249-98f00699-b8fc-46bb-a2e6-e3812410c6d1.png)


![Screenshot from 2023-01-04 23-58-11](https://user-images.githubusercontent.com/65233860/210672268-e740b41e-705f-4edc-ad31-68a2307ddffc.png)


![Screenshot from 2023-01-04 23-59-06](https://user-images.githubusercontent.com/65233860/210672283-306126b4-efd8-4d7b-b5cc-3ac1043cb926.png)


![Screenshot from 2023-01-04 23-59-44](https://user-images.githubusercontent.com/65233860/210672307-9a9b3217-c2b1-413e-8fbd-5b6b726afcff.png)